### PR TITLE
Ice beam gate room temporary blue

### PR DESCRIPTION
**Pull request by blkerby**

### CODE REVIEW

**Comment by blkerby** [comment]
Nito found this one, did it on stream a while ago.

File: region/norfair/west/Ice Beam Gate Room.json [hunk 1]
```diff
@@ -359,6 +359,70 @@
...
+          "length": 4,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canChainTemporaryBlue"
+      ],
+      "clearsObstacles": ["A", "B"],
+      "note": [
+        "Use a temporary blue chain to jump through the bomb blocks and land on the center of the ledge above.",
```
Comment (osse101) [inline region/norfair/west/Ice Beam Gate Room.json:376]: I'd rather not land on the ledge above and go directly for the bomb block.
If you do land above, this needs can4Highmidairmorph.

File: region/norfair/west/Ice Beam Gate Room.json [hunk 2]
```diff
@@ -359,6 +359,70 @@
...
+        {"or": [
+          {"canShineCharge": {
+            "usedTiles": 14,
+            "openEnd": 0
+          }},
+          {"and": [
+            {"canShineCharge": {
+              "usedTiles": 15,
+              "openEnd": 0
+            }},
```
Comment (osse101) [inline region/norfair/west/Ice Beam Gate Room.json:412]:
```suggestion
            }},
            "can4HighMidAirMorph",
```

**Review by osse101 — COMMENTED** [review]
[4, 5] has a use in setting up GMode.

The jump into the 1 tile bomb block is pretty hard.  I don't like it.

File: region/norfair/west/Ice Beam Gate Room.json (hunk 1 above)
Comment (blkerby) [reply region/norfair/west/Ice Beam Gate Room.json:376]: I found going directly for the bomb block to be much more difficult. Just checked with Bizhawk and confirmed that doing it that way requires a frame-perfect morph (at least from the position I tested), whereas from the ledge there is a 2-frame window. Adding can4Highmidairmorph sounds good.

**Pull request merged**